app/assets/stylesheets/application.scss

```diff
@@ -146,12 +154,15 @@ h5 {
 }
 
 .profile-photo {
+  width: 100%;
+  border-radius: 50%;
   margin-top: 15px;
 }
 
 .msw-swa /* Inherits from swell arrows */
 {
-   background: url("http://im-1.msw.ms/md/static/sa-sprite.png") no-repeat top left;
+  display: inline-block;
+  background: url("http://im-1.msw.ms/md/static/sa-sprite.png") no-repeat top left;
```

Comment:
URL `http://im-1.msw.ms/md/static/sa-sprite.png` should not contain protocol or domain